src/App.js

```diff
@@ -15,6 +15,7 @@ import { availableLanguageOrNull } from './i18n/utils';
 import Shopify from './components/Integrations/Shopify/Shopify';
 import SignupConfirmation from './components/Signup/SignupConfirmation';
 import SubscriberHistory from './components/Reports/SubscriberHistory/SubscriberHistory';
+import SubscriberHistorySentCampaigns from './components/Reports/SubscriberHistory/SubscriberHistoryCurrentSearch/SubscriberHistorySentCampaigns/SubscriberHistorySentCampaigns';
```

Comment:
I'm not so sure about this, since now we can access via url, it's an independent component now, outside search. What are you thoughts?

Reply:
yes!